packages/medusa/src/services/order-edit.ts

```diff
@@ -0,0 +1,50 @@
+import { EntityManager } from "typeorm"
+import { MedusaError } from "medusa-core-utils"
+import { OrderEditRepository } from "../repositories/order-edit"
+import { OrderEditStatus } from "../models/order-edit"
+import { TransactionBaseService } from "../interfaces"
+
+type InjectedDependencies = {
+  manager: EntityManager
+  orderEditRepository: typeof OrderEditRepository
+}
+
+export class OrderEditService extends TransactionBaseService {
+  protected transactionManager_: EntityManager | undefined
+  protected readonly manager_: EntityManager
+
+  protected readonly orderEditRepository_: typeof OrderEditRepository
+
+  constructor({ manager, orderEditRepository }: InjectedDependencies) {
+    // eslint-disable-next-line prefer-rest-params
+    super(arguments[0])
+
+    this.manager_ = manager
+    this.orderEditRepository_ = orderEditRepository
+  }
+
+  async delete(orderEditId: string): Promise<void> {
+    return await this.atomicPhase_(async (manager) => {
+      const orderEditRepo = manager.getCustomRepository(
+        this.orderEditRepository_
+      )
+
+      const edit = await orderEditRepo.findOne({ where: { id: orderEditId } })
+
+      if (!edit) {
+        return
```

Comment:
thoughts(non-blocking): should we return a 404 here if there is no order edit corresponding to the id, so that the user does not think that the deletion occurred while the order edit does not exists?